File: Examples/Ruby/DrawingObjects/get_document_drawing_object_ole_data.rb
require 'aspose_words_cloud'

class DrawingObjects

  include AsposeWordsCloud
  include AsposeStorageCloud

  def initialize
    #Get App key and App SID from https://cloud.aspose.com
    AsposeApp.app_key_and_sid("APP_KEY", "APP_SID")
    @words_api = WordsApi.new
  end

  def upload_file(file_name)
    @storage_api = StorageApi.new
    response = @storage_api.put_create(file_name, File.open("../data/" << file_name,"r") { |io| io.read } )
  end

  def get_document_drawing_object_ole_data
    file_name = "Sample_EmbeddedOLE.docx"
    upload_file(file_name)

    object_index = 0
    response = @words_api.get_document_drawing_object_ole_data(file_name, object_index)
  end

end

drawingObjects = DrawingObjects.new()
puts drawingObjects.get_document_drawing_object_ole_data

File: Examples/PHP/DrawingObjects/GetDocumentDrawingObjectOleData.php
<?php
require_once realpath(__DIR__ . '/..') . '/vendor/autoload.php';
require_once realpath(__DIR__ . '/..') . '/Utils.php';

use Aspose\Words\WordsApi;
use Aspose\Words\AsposeApp;

class DrawingObjects {

	public $wordsApi;

	public function __construct() {
		AsposeApp::$appSID = Utils::appSID;
		AsposeApp::$apiKey = Utils::apiKey;
		$this->wordsApi = new WordsApi();
	}

	public function getDocumentDrawingObjectOleData() {
		// Upload file to Aspose Cloud Storage
		$fileName = "Sample_EmbeddedOLE.docx";
		Utils::uploadFile($fileName);

        $result = $this->wordsApi->GetDocumentDrawingObjectOleData($fileName, $objectIndex=0, $storage = null, $folder = null);        
        print_r($result);
	}
}

$drawingObjects = new DrawingObjects();
$drawingObjects->getDocumentDrawingObjectOleData();

?>